### ISSUE TITLE
Entering 1.130 in the product edit form in Spanish should be 1130

### ISSUE THREAD

**Issue by stephanegigandet** [openfoodfacts/openfoodfacts-server#3560]
We currently turn it into 1.13

We should do it at least for the energy field in kJ. It might also happen for nutrition facts in mg when the value is more than 1000 mg.

There are certainly other languages that have the same issue, it needs investigation.

![image](https://user-images.githubusercontent.com/8158668/83942079-dffb4f00-a7f0-11ea-905a-3c555f03cfac.png)

https://es.openfoodfacts.org/cgi/product.pl?type=edit&code=8480000171474

**Comment by VaiTon** [comment]
Same for it, we use the dot as separation between thousands. eg 1.000,00

**Comment by hangy** [comment]
> There are certainly other languages that have the same issue, it needs investigation.

Yup, see https://github.com/patch/cldr-number-pm5/blob/249762a48c7daa1e66ad64f113c5dce807563824/lib/CLDR/Number/Data/Base.pm for `group => '.'`
